### PR TITLE
Include dist folder to upload to npm and babelify interpretation-player

### PR DESCRIPTION
**Pull request by deevannaveed**
---
Fix dist folder not found error in rsi-interpretation-player module and transpile it to fix a bug related to react
---

The purpose of this PR is to fix the following bugs related to interpretation-player. 
1. Consumers of our package will not find the dist directory and the solution is to include that directory in the package.json file.
2. There was a bug with the newer versions of React when it could not compile the es6 to es5 so we have to do it on our side and the solution was to use the `getBabelOutputPlugin` which will post process the script that was generated by rollup and transpile it.
3. There was `s` missing at the end of props(`hasSubtitles`) of React and Vue examples and that's why the subtitles were not working.

## How To Test
Since Abel will be the reviewer so I will send you the zip file of all the three flavors(react, vue, vanilljs) and you can run them locally, if they work with subtitles it means everything is fine.
### Credentials
Domains https://akkadu.com and https://rsi.akkadu.com
Event: `Naveed Experiment`
EO: `naveed@akkadu-team.com`
Password: `testtest56`

## Checklist

- [x] My code follows the code style of this project.
- [x] I have removed all unused css, language strings, variables, functions, images and other resources.
- [ ] My change requires to update dependencies and I have updated `package.json` and `yarn.lock` accordingly.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

加油！

### CODE REVIEW

File: packages/rsi-interpretation-player/rollup.config.js [hunk 1]
```diff
@@ -18,7 +18,13 @@ export default {
     inlineDynamicImports: true,
   },
   plugins: [
-    babel(),
+    commonjs(),
+    getBabelOutputPlugin({
+      presets: ['@babel/preset-env'],
+      plugins: [
+        ["@babel/transform-runtime"]
+      ]
```
Comment (abelbordonado) [inline packages/rsi-interpretation-player/rollup.config.js:26]: Thats the imporant part right 💪 
Cool, I can select several lines while doing a comment (just realized)

Comment (deevannaveed) [reply packages/rsi-interpretation-player/rollup.config.js:26]: Yes and to include the `./dist` in package.json.

**Comment by abelbordonado** [comment]
Comments zip:
 - React and Vainilla: Would be nice to copy Vue style (which centers and top margin the dropdown) 
 - Readme: React default readme is so long xD, vue is good but both will be great to personalize a little about akkadu and just mention the `yarn start` required. Vainilla would mention to use `serve` tu run it

**Review by abelbordonado — APPROVED** [review]
Subtitles work great and are smooth in all the flavors! 
I made some tiny comments for the example zip.
This branch it self can be merged :)

**Pull request merged**